src/PaymentGateways/Gateways/Stripe/Migrations/AddStatementDescriptorToStripeAccounts.php

```diff
@@ -27,7 +30,14 @@ public function run()
             foreach ($allStripeAccount as $index => $stripAccount) {
                 if (!isset($stripAccount['statement_descriptor'])) {
                     $statementDescriptor = trim($statementDescriptor);
-                    $this->validateStatementDescriptor($statementDescriptor);
+
+                    try {
+                        $this->validateStatementDescriptor($statementDescriptor);
+                    } catch (Exception $e) {
+                        $unsupportedCharacters = ['<', '>', '"', '\''];
```

Comment:
This filtering logic is duplicated in the `AccountDetail` model and should be extracted. Also, this `catch` only handles unsupported characters and max length, but does not account for a numeric value or a min length.